src/snowflake/cli/plugins/snowpark/package_utils.py

```diff
@@ -139,11 +141,11 @@ def download_unavailable_packages(
         )
 
     # download all packages with their dependencies
-    with Venv() as v, SecurePath.temporary_directory() as downloads_dir:
+    with SecurePath.temporary_directory() as downloads_dir:
         # This is a Windows workaround where use TemporaryDirectory instead of NamedTemporaryFile
-        requirements_file = SecurePath(v.directory.name) / "requirements.txt"
+        requirements_file = SecurePath("requirements.txt")
```

Comment:
Build the path using `downloads_dir` to avoid loosely coupled paths.

Reply:
Done